### DISCUSSION TITLE
Is it possible to answer the new App Privacy Questions with fastlane?

### DISCUSSION THREAD

**Discussion by JimHowell20** [fastlane/fastlane#17625] [General] [answered]
Edit: nevermind, I found the discussion about this!

**Comment by max-ott** [comment]
Would be helpful to post at least a link in there 🤷‍♂️

**Reply by JimHowell20** [reply]
https://github.com/fastlane/fastlane/discussions/17467

**Comment by rogerluan** [comment, marked as answer]
Full docs were released here 🚀 https://docs.fastlane.tools/uploading-app-privacy-details/
